src/ui/public/angular-bootstrap/datepicker/day.html

```diff
@@ -1,20 +1,82 @@
-      <th><button type="button" class="btn btn-default btn-sm pull-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
-      <th colspan="{{5 + showWeeks}}"><button id="{{uniqueId}}-title" role="heading" aria-live="assertive" aria-atomic="true" type="button" class="btn btn-default btn-sm" ng-click="toggleMode()" tabindex="-1" style="width:100%;"><strong>{{title}}</strong></button></th>
-      <th><button type="button" class="btn btn-default btn-sm pull-right" ng-click="move(1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-right"></i></button></th>
+      <th
+        class="kuiDatePickerNavigationCell"
+        colspan="{{7 + showWeeks}}"
+      >
+        <div class="kuiDatePickerNavigation">
+          <button
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="move(-1)"
+            tabindex="-1"
+          >
+            <span class="kuiIcon fa-chevron-left"></span>
+          </button>
+
+          <button
+            id="{{uniqueId}}-title"
+            role="heading"
+            aria-live="assertive"
+            aria-atomic="true"
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="toggleMode()"
+            tabindex="-1"
+            style="width:100%;"
+          >
+            <strong>{{title}}</strong>
+          </button>
+
+          <button
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="move(1)"
+            tabindex="-1"
+          >
+            <span class="kuiIcon fa-chevron-right"></span>
+          </button>
+        </div>
+      </th>
     </tr>
     <tr>
       <th ng-show="showWeeks" class="text-center"></th>
-      <th ng-repeat="label in labels track by $index" class="text-center"><small aria-label="{{label.full}}">{{label.abbr}}</small></th>
+      <th ng-repeat="label in labels track by $index" class="text-center">
+        <small aria-label="{{label.full}}">{{label.abbr}}</small>
+      </th>
     </tr>
   </thead>
   <tbody>
     <tr ng-repeat="row in rows track by $index">
-      <td ng-show="showWeeks" class="text-center h6"><em>{{ weekNumbers[$index] }}</em></td>
-      <td ng-repeat="dt in row track by dt.date" class="text-center" role="gridcell" id="{{dt.uid}}" aria-disabled="{{!!dt.disabled}}">
-        <button type="button" style="width:100%;" class="btn btn-default btn-sm" ng-class="{'btn-info': dt.selected, active: isActive(dt)}" ng-click="select(dt.date)" ng-disabled="dt.disabled" tabindex="-1"><span ng-class="{'text-muted': dt.secondary, 'text-info': dt.current}">{{dt.label}}</span></button>
+      <td ng-show="showWeeks" class="kuiDatePickerHeaderCell">
+        <em>{{ weekNumbers[$index] }}</em>
+      </td>
+
+      <td
+        ng-repeat="dt in row track by dt.date"
+        class="kuiDatePickerRowCell"
+        ng-class="{'kuiDatePickerRowCell-isPrecluded': dt.disabled && !dt.secondary}"
```

Comment:
The angular-bootstrap calendar is weird, in that you can't select a day on the "From" calendar if it falls after the selected day in the "To" calendar (and vice versa, you can't select a "To" day if it is before the "From" day). So in this sense, those unavailable days are "precluded" by the selection on the other calendar. I thought it was important to distinguish this from a simple "disabled" state since you can actually disable days for other reasons (e.g. if a day falls on the previous or following month). So that's why I chose "precluded". What do you think? Does this name communicate the intent, or can you think of a name that makes more sense? Open to suggestions. :)